drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/handlers/ScenarioSimulationGridPanelClickHandlerTest.java

```diff
@@ -130,56 +130,56 @@ public void setScenarioGrid() {
 
     @Test
     public void setOtherContextMenu() {
-        scenarioSimulationGridPanelClickHandler.setOtherContextMenu(mockOtherContextMenu);
-        assertEquals(mockOtherContextMenu, scenarioSimulationGridPanelClickHandler.otherContextMenu);
+        scenarioSimulationGridPanelClickHandler.setOtherContextMenu(otherContextMenuMock);
+        assertEquals(otherContextMenuMock, scenarioSimulationGridPanelClickHandler.otherContextMenu);
     }
 
     @Test
     public void setHeaderGivenContextMenu() {
-        scenarioSimulationGridPanelClickHandler.setHeaderGivenContextMenu(mockHeaderGivenContextMenu);
-        assertEquals(mockHeaderGivenContextMenu, scenarioSimulationGridPanelClickHandler.headerGivenContextMenu);
+        scenarioSimulationGridPanelClickHandler.setHeaderGivenContextMenu(headerGivenContextMenuMock);
+        assertEquals(headerGivenContextMenuMock, scenarioSimulationGridPanelClickHandler.headerGivenContextMenu);
     }
 
     @Test
     public void setHeaderExpectedContextMenu() {
-        scenarioSimulationGridPanelClickHandler.setHeaderExpectedContextMenu(mockHeaderExpectedContextMenu);
-        assertEquals(mockHeaderExpectedContextMenu, scenarioSimulationGridPanelClickHandler.headerExpectedContextMenu);
+        scenarioSimulationGridPanelClickHandler.setHeaderExpectedContextMenu(headerExpectContextMenuMock);
+        assertEquals(headerExpectContextMenuMock, scenarioSimulationGridPanelClickHandler.headerExpectedContextMenu);
```

Comment:
I like having the mock at the end. Easier to read.

Reply:
Yup. I'm going to apply this name convention on all mocked objects